cli/azd/cmd/down.go

```diff
@@ -97,30 +98,34 @@ func newDownAction(
 }
 
 func (a *downAction) Run(ctx context.Context) (*actions.ActionResult, error) {
-	infraManager, err := provisioning.NewManager(
-		ctx,
-		a.env,
-		a.projectConfig.Path,
-		a.projectConfig.Infra,
-		a.console.IsUnformatted(),
-		a.azCli,
-		a.console,
-		a.commandRunner,
-		a.accountManager,
-		a.userProfileService,
-		a.subResolver,
-		a.alphaFeatureManager,
-	)
-
+	// silent manager for running Plan()
+	infraManager, err := createProvisioningManager(ctx, a, &project.MutedConsole{ParentConsole: a.console})
```

Comment:
If we're surfacing `MutedConsole`, then it probably should move to `output` package instead of `project`. Can be a separate PR.